cabby/model/text/dual_encoder/train.py

```diff
@@ -88,6 +88,8 @@ def evaluate(self, validation_set: bool = True):
         cellids = batch['cellid'].float().to(self.device)
         neighbor_cells = batch['neighbor_cells'].float().to(self.device) 
         far_cells = batch['far_cells'].float().to(self.device)
+        distibution = batch['distribution'].to(self.device)
```

Comment:
sp: distribution

Reply:
Also, should this be distance_distribution?

Reply:
changed to distance_distribution